src/services/loginHistory.ts

```diff
@@ -29,6 +29,7 @@ export async function findLoginHistoriesByUserID(
     ])
     .from('login_history')
     .where('user_id', userID)
+    .limit(50)
```

Comment:
これ、引数でOptionalで渡したい気がする。

```suggestion
    .limit(limit ?? 50)
```